src/audio_call.c

```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
     print_err (self, error_str);
 }
 
+void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[MAX_STR_SIZE])
+{
+    char *error_str;
+
+    if ( argc != 1 ) {
+        error_str = "Must have value!";
+        goto on_error;
+    }
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = atoi(argv[1]);
```

Comment:
Error checking is done through the switch at the bottom (bounds checking between 6 and 510). Should I add a second check inside of the main function that also prints "Invalid audio bit rate value"?